src/DopplerVelocityLog.cc

```diff
@@ -1731,12 +1731,34 @@ namespace gz
               samplePointInSensorFrame;
 
           // Transform sample point to the environmental data frame
-          const gz::math::Vector3d samplePointInDataFrame =
+          std::optional<gz::math::CoordinateVector3>
```

Comment:
This can still be const.

Reply:
made `const`. [18faa46](https://github.com/gazebosim/gz-sensors/pull/460/commits/18faa46c3d390f532dde6b934d0d100837d51fa3)